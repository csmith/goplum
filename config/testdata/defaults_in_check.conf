# Test that defaults blocks cannot be nested inside check blocks
check http.get "test-site" {
  url = "https://example.com"
  interval = 30s
  
  # This should not be allowed
  defaults {
    timeout = 10s
  }
}