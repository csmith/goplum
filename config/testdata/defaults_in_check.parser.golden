"unexpected defaults keyword (defaults) at line 5 column 33, expecting one of: end of block, identifier"